opm/input/eclipse/EclipseState/Aquifer/NumericalAquifer/SingleNumericalAquifer.cpp

```diff
@@ -26,6 +26,11 @@
 #include <opm/input/eclipse/EclipseState/Aquifer/NumericalAquifer/SingleNumericalAquifer.hpp>
 #include "../AquiferHelpers.hpp"
 
+#include <opm/common/OpmLog/OpmLog.hpp>
+
+#include <fmt/format.h>
+#include <algorithm>
```

Comment:
Due to the code update, we might not need the headers <algorithm> and <limits> in this file now. 

<limits> is needed by the file [NumericalAquifers.cpp](https://github.com/OPM/opm-common/pull/4359/files#diff-b15cc1962316f440a8a9a69fc62c69195a3738a79912d0363938380714debf23)